package-lock.json

```diff
@@ -95,6 +95,7 @@
 				"nanoid": "^3.3.4",
 				"nest-winston": "^1.8.0",
 				"nestjs-console": "^8.0.0",
+				"node-html-parser": "^5.4.2-0",
```

Comment:
why this package-lock change?
use npm ci, and if needed to update the version, should be reflected there as well.

Reply:
I delete this lib in last commit. Cedric doesn't want an additional lib. So I removed it and use an existing lib.

Reply:
Ok, but it must also remove from the package-lock. For this it is the best to use npm-uninstall. Otherwise it still alive in the lock file and get unexpected results if anyone execute npm ci. Add it again and remove it over save way should fix this without removing the entry in the lock by your self.

Reply:
Done in last commit.